test/tests/functional/pbs_resource_usage_log.py

```diff
@@ -55,18 +55,27 @@ def setUp(self):
         attr2 = {'resources_available.mem': '200gb'}
         self.server.manager(MGR_CMD_SET, NODE, attr2, id=self.mom.shortname)
 
+    def cleanup_eatcpu(self, scripts):
```

Comment:
Shouldn't the eat cpu script be cleaned up by the mom?  It's submitted as part of the job script.  If it isn't, that is a bug in mom, not the test.

Reply:
@bhroam You are correct. eatcpu script is cleaned up by mom. Only where the test case SIGKILL mom there the cleanup is not happening. So made changes to only those tests.